spyder/widgets/sourcecode/api/decorations.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
It was @goanpeca idea to create another api just for the editor, I'm not sure where to put this, maybe `sourcecode/manager.py` will be better, I don't like the pidea opf put code in `__init__.py`